### PR TITLE
Modify User Guide

### LINKED ISSUE #164: Update on UG v2 Wei Hao

### PR DESCRIPTION
**Pull request by chewwh09**
1. Update the Reminder feature
2. Move the delete command below the list and find commands. Reason: Delete has an example that uses the list and find command.
3. Rearrange the command summary to follow the sequence of features.
4. Update the command summary.
4. Format the second QnA format problem.
5. Correct the format of StoreMando's layout section
6. Remove all the </br>
7. Remove the dashed lines in section 1.2

closes #164

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -38,13 +38,10 @@ If you need an overview of StoreMando’s commands, check out the [“Command Su
 Please note the following symbols used in the User Guide which may serve as points of interests:
 
 `command` | The grey highlight indicates commands that can be executed by StoreMando.
---------|------------------
 
 :information_source: | This symbol indicates information that you may wish to take note.
----------------------|------------------
 
 :bulb: | This symbol indicates tips provided by us.
```
Comment (JayChenYJ) [inline docs/UserGuide.md:44]: are the lines required for proper display? oTHER :bulb: etc got such lines?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -69,14 +66,14 @@ Please note the following symbols used in the User Guide which may serve as poin
 
 ### 2.2. StoreMando's layout
 
-1. Main Panel
+1. Main Panel <br>
```
Comment (JayChenYJ) [inline docs/UserGuide.md]: If you are putting a line break then i suggest you give it #### for the sub headers :)

File: docs/UserGuide.md [hunk 3]
```diff
@@ -291,9 +287,11 @@ Format: `reminder NUMBER TIME_UNIT`
 
 **:information_source: Notes about the reminder command:**<br>
 
-* `TIME_UNIT` must be either `days` or `weeks`. `day` and `week` **will not work**.
+* `NUMBER` **can be any integer** …​, -7, -3, 0, 3, 7, …​
+* `TIME_UNIT` must be either `day(s)` or `week(s)`.
+  * `NUMBER` in the range of **[-1, 0, 1]** must have`TIME_UNIT` to be **singular**
```
Comment (JayChenYJ) [inline docs/UserGuide.md]: This one no need already. Can change to 
`TIME_UNIT` must be either `days` or `weeks`.
`day` or `week` accepted when number is in the range of **[-1, 0, 1]**

File: docs/UserGuide.md [hunk 4]
```diff
@@ -304,6 +302,14 @@ Examples:
...
   * `reminder 5 days` returns the chocolate only.
   * `reminder 2 weeks` returns both the chocolate and the banana.
+  <br>
+* `reminder -7 days` returns a list containing all items that has been expired for 7 days.
+* `reminder -2 weeks` returns a list containing all items that has been expired for 2 weeks.
+* Given today is 30 March 2021, and the inventory has 2 items: apple with expiry date of 27 March 2021 and
+  banana with an expiry date of 20 March 2021.
+  * `reminder -5 days` returns the apple only.
+  * `reminder -2 weeks` returns both the apple and the banana.
+
```
Comment (JayChenYJ) [inline docs/UserGuide.md:312]: This example abit too specific, by the time people read it it won't make sense already

File: docs/UserGuide.md (hunk 3 above)
Comment (chewwh09) [reply docs/UserGuide.md]: All right noted.

File: docs/UserGuide.md (hunk 2 above)
Comment (chewwh09) [reply docs/UserGuide.md]: All right noted!

File: docs/UserGuide.md (hunk 1 above)
Comment (chewwh09) [reply docs/UserGuide.md:44]: Yes they do!

File: docs/UserGuide.md (hunk 4 above)
Comment (chewwh09) [reply docs/UserGuide.md:312]: But it was stated that "given today is 30 March".

Solved through other platform.

**Review by JayChenYJ — APPROVED** [review]
LGTM!

**Pull request merged**